packages/ndla-ui/src/Figure/Figure.js

```diff
@@ -89,42 +95,69 @@ FigureCaption.propTypes = {
     external: PropTypes.bool,
   }),
   locale: PropTypes.string,
+  hideFigcaption: PropTypes.bool,
 };
 
 FigureCaption.defaultProps = {
   link: null,
 };
 
-export const Figure = ({
-  children,
-  type,
-  resizeIframe,
-  noFigcaption,
-  ...rest
-}) => {
-  const typeClass = `u-float-${type}`;
+class Figure extends Component {
+  constructor(props) {
+    super();
+    this.state = {
```

Comment:
Du kan ikke bruke state eller noe dynamisk React funksjonalitet for innhold i artikler. Det må gjøres via plain javascript i ndla-article-scripts.

Reply:
done